src/inference/src/system_conf.cpp

```diff
@@ -459,6 +465,11 @@ int get_num_sockets() {
     return cpu_info()._sockets;
 }
 
+int numa_node_id(int cpu_id) {
```

Comment:
minor: `numa_node_id` -> `get_numa_node_id` to align with other functions

Reply:
Ok, done.